app/src/main/java/app/revanced/manager/ui/screen/settings/AdvancedSettingsScreen.kt

```diff
@@ -167,11 +176,10 @@ fun AdvancedSettingsScreen(
             )
             val clipboard = remember { context.getSystemService<ClipboardManager>()!! }
             val deviceContent = """
-                    Version: ${BuildConfig.VERSION_NAME} (${BuildConfig.VERSION_CODE})
-                    Build type: ${BuildConfig.BUILD_TYPE}
+                    Version: ${BuildConfig.VERSION_NAME}, ${BuildConfig.BUILD_TYPE} (${BuildConfig.VERSION_CODE})
                     Model: ${Build.MODEL}
                     Android version: ${Build.VERSION.RELEASE} (${Build.VERSION.SDK_INT})
-                    Supported Archs: ${Build.SUPPORTED_ABIS.joinToString(", ")}
```

Comment:
Why not "Supported Architectures"?

Reply:
Because ARMv7 is no longer an issue anymore, I assume

Reply:
The reason it was changed to Preferred Architecture at the time because it's accurate since it was sorted based on most preferred to least preferred.